extensions/amp-story/1.0/simple-template.js

```diff
@@ -79,13 +79,27 @@ function renderSingle(doc, elementDef) {
       )
     : doc.createElement(elementDef.tag);
 
-  if (hasOwn(elementDef, 'localizedStringId')) {
+  const hasLocalizedTextContent = hasOwn(elementDef, 'localizedStringId');
+  const hasLocalizedLabel = hasOwn(elementDef, 'localizedLabelId');
+  if (hasLocalizedTextContent || hasLocalizedLabel) {
     const win = toWin(doc.defaultView);
     Services.localizationServiceForOrNull(win).then(localizationService => {
       devAssert(localizationService, 'Could not retrieve LocalizationService.');
-      el.textContent = localizationService.getLocalizedString(
-        /** @type {!LocalizedStringId} */ (elementDef.localizedStringId)
-      );
+
+      if (hasLocalizedTextContent) {
+        el.textContent = localizationService.getLocalizedString(
+          /** @type {!LocalizedStringId} */ (elementDef.localizedStringId)
+        );
+      }
+
+      if (hasLocalizedLabel) {
+        el.setAttribute(
+          'aria-label',
+          localizationService.getLocalizedString(
+            /** @type {!LocalizedStringId} */ (elementDef.localizedLabelId)
```

Comment:
Can you update the typedef `ElementDef` above to add the `localizedLabelId` property?

(annoyingly, GitHub doesn't let reviewers comment on lines of code that are too far from what the author has changed...)

Reply:
Updated (and learned a bit about how the typing system works)!